src/main/java/org/jabref/logic/importer/fileformat/BibtexParser.java

```diff
@@ -389,6 +393,13 @@ private void parseJabRefComment(Map<String, String> meta) {
         }
     }
 
+    @VisibleForTesting
+    Optional<JsonObject> parseCommentToJson(String comment) {
```

Comment:
This is not embedded in a real parsing of a future JabRef version. This should be done in this PR too so that we are sure about the architcture.

(That comment was risen, because not a pattern like `String rest = comment.substring(MetaData.META_FLAG.length());` was used, but an offset...)